README.md

```diff
@@ -1,4 +1,263 @@
+    poorly-received.
+  - Submit a pull request. As a pull request the RFC will receive design
+    feedback from the larger community, and the author should be prepared to respond to feedback and possibly revise the proposal.
+
+### Status: Review
+
+Once a an RFC has been submitted, it's ready for Review.
+
+  - Each pull request will be labeled with the most relevant [working group], which
+    will lead to its being triaged by that team in a future meeting and assigned
+    to a member of the working group.
+  - At this point, the status in the RFC markdown document should be marked "Review"
+  - Build consensus and integrate feedback. RFCs that have broad support are
+    much more likely to make progress than those that don't receive any
+    comments. Feel free to reach out to the RFC assignee in particular to get
+    help identifying stakeholders and obstacles.
+  - The working group will discuss the RFC pull request, as much as possible in the
+    comment thread of the pull request itself. Offline discussion must be
+    summarized on the pull request comment thread.
+  - RFCs rarely go through this process unchanged, especially as alternatives
+    and drawbacks are shown. You can make edits, big and small, to the RFC to
+    clarify or change the design, but make changes as new commits to the pull
+    request, and leave a comment on the pull request explaining your changes.
+    Specifically, do not squash or rebase commits after they are visible on the
+    pull request.
+
+### Status: Final Call (FCP)
+
+At some point, a member of the working group will propose a motion for _"final
+comment period"_ (FCP), along with a *disposition* for the RFC (merge, close,
+    or postpone).
+
+  - This step is taken when enough of the tradeoffs have been discussed that
+  the working group is in a position to make a decision. That does not require
+  consensus amongst all participants in the RFC thread (which is usually
+  impossible). However, the argument supporting the disposition on the RFC
+  needs to have already been clearly articulated, and there should not be a
+  strong consensus *against* that position outside of the working group. Working group
+  members use their best judgment in taking this step, and the FCP itself
+  ensures there is ample time and notification for stakeholders to push back
+  if it is made prematurely.
+  - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
+    a *summary comment* trying to lay out the current state of the discussion
+    and major tradeoffs/points of disagreement.
+  - Before actually entering FCP, *all* members of the working group must sign off;
+  this is often the point at which many working group members first review the RFC
+  in full depth.
+- The FCP lasts ten calendar days, so that it is open for at least 5 business
+  days. It is also advertised widely,
+  e.g. in the [Weekly IPFS All Hands Call](https://github.com/ipfs/pm/#weekly-all-hands). This way all
```

Comment:
Maybe a personal preference of mine, but I think any broadcasts, such as status changes of RFCs, should be announced on an RFC specific mailing list so that they don't get lost in meeting notes.